README.md

```diff
@@ -45,6 +59,7 @@ This is done by implementing the `application:openURL:sourceApplication:annotati
 }
 ```
 
+### Log in with Clever Button
 You can optionally add the Clever Instant Login button.
 In the `UIViewController` where you set the login success/failure handlers, add the button:
```

Comment:
Now that we're putting the success/failure handlers in AppDelegate, should we update this recommendation too?